manifests/charts/istio-control/istio-discovery/files/injection-template.yaml

```diff
@@ -211,6 +211,10 @@ spec:
           - wait
   {{- end }}
     env:
+    {{- if eq (env "PILOT_ENABLE_INBOUND_PASSTHROUGH" "true") "false" }}
+    - name: REWRITE_PROBE_LEGACY_LOCALHOST_DESTINATION
```

Comment:
It seems this is not necessary.

Reply:
Why not? its for backwards compatibility. If they have localhost listeners, they can set PILOT_ENABLE_INBOUND_PASSTHROUGH on pilot env temporarily and retain existing behavior